addon/mixins/dynamic-properties.js

```diff
@@ -0,0 +1,267 @@
+/**
+  @module ember-flexberry-gis
```

Comment:
Please, change namespce into 'ember-flexberry' instead of 'ember-flexberry-gis'.